src/Trixi.jl

```diff
@@ -81,7 +81,8 @@ export initial_condition_constant,
        initial_condition_sedov_blast_wave, initial_condition_medium_sedov_blast_wave,
        initial_condition_blob,
        initial_condition_orszag_tang,
-       initial_condition_rotor
+       initial_condition_rotor,
+       initial_condition_taylor_green_vortex
```

Comment:
If you `export` this IC here, you should also change the line
```julia
initial_condition = Trixi.initial_condition_taylor_green_vortex
```
in `examples/3d/elixir_euler_taylor_green_vortex.jl`.

Reply:
Thanks! Fixed in dfa3b2fc.